src/arr/trove/statistics.arr

```diff
@@ -241,3 +267,78 @@ fun r-squared(x :: List<Number>, y :: List<Number>, f :: (Number -> Number)) ->
   end
 end
 
+fun z-test(sample-list :: List, population-sd :: Number, population-mean :: Number) -> Number:
+  doc: "z-test"
```

Comment:
These docstrings don't add any info; either fix or remove... (here and throughout the file)

Reply:
Done³. I have already noted (to @schanzer) that the function names should probably use `score` rather than `test`, but I'm going by the spec he got from his SME(s)